int64 iden
---
Object[] obj_lst
int64 iden
